src/components/common/HighlightsView.vue

```diff
@@ -503,29 +466,29 @@ export default defineComponent({
             }
           }
         )
-        // TODO catch errors in response
+        // TODO mavedb-ui#130 catch errors in response
         return response.data || {}
       } catch (err) {
-        console.log(`Error while loading search results")`, err)
+        console.log(`Error while loading search results for Model: ${model}, Name: ${name}, Field: ${field}")`, err)
         return []
       }
     },
 
-    searchDoiIdentifiers: async function (identifier) {
+    searchIdentifiers: async function (identifier, field) {
       try {
         const response = await axios.post(
-          `${config.apiBaseUrl}/doi-identifiers/search`,
+          `${config.apiBaseUrl}/${field}/search`,
           { text: identifier },
           {
             headers: {
               accept: 'application/json'
             }
           }
         )
-        // TODO catch errors in response
+        // TODO mavedb-ui#130 catch errors in response
```

Comment:
does github like this formatting? for some reason I had it in my head that we should use TODO(#130) for issues in the same repo

Reply:
Yeah that format is better, idk why I thought I needed the repo when obviously it would be in the UI repo lol. Fixed